src/main/src/main/java/org/geoserver/catalog/CatalogBuilder.java

```diff
@@ -515,6 +518,10 @@ public void setupMetadata(FeatureTypeInfo ftinfo, FeatureSource featureSource)
     public ReferencedEnvelope getLatLonBounds(ReferencedEnvelope nativeBounds,
             CoordinateReferenceSystem declaredCRS) throws IOException {
         if (nativeBounds != null && declaredCRS != null) {
+            // shortcut: return empty envelope if nativeBounds is empty
+            if (nativeBounds.isEmpty()) {
+                return new ReferencedEnvelope(nativeBounds, DefaultGeographicCRS.WGS84);
```

Comment:
Hi Stefano, wondering, is we go this way, are the bounds published in WFS valid schema wise? 
Normally for geometryless data I still keep a valid bbox, e.g., maxx > minx, maxy > miny